main.js

```diff
@@ -0,0 +1,83 @@
+const theremin = document.getElementById("theremin");
+const form = document.querySelector(".form");
+const maxInput = document.getElementById("max");
+const minInput = document.getElementById("min");
+const submitBtn = document.getElementById("submit");
+
+const AudioContext = window.AudioContext || window.webkitAudioContext;
+const audioCtx = new AudioContext();
+const gainNode = audioCtx.createGain();
+let oscillator, mousedown, minValue, maxValue;
+
+// Helper functions
+const calculateFrequency = (Xpos) => {
+	let min = minValue ? minValue : 10,
+		max = maxValue ? maxValue : 1000;
```

Comment:
I have two comments about this part:
- A simpler syntax to replace 
```javascript
let min = minValue ? minValue : 10
```
could be:
```javascript
let min = minValue || 10
```
- In my opinion, you should when possible avoid using global values like in this case **minValue** and **maxValue**. I had to look up where they were defined and that took me some time to figure out where they were, and where are they modified. This could seem like a good approach, but when you begin working on bigger projects, global variables should be avoided like plague, so try to get used to avoiding them once and for all.

Reply:
Thank you, I added a comment on my code, for myself to remember that! =)